scripts/tests/chiptest/accessories.py

```diff
@@ -33,141 +34,162 @@
+        if accessory := self._accessories[name]:
             # The args param comes directly from the sys.argv[2:] of Start.py and should contain a list of strings in
             # key-value pair, e.g. [option1, value1, option2, value2, ...]
-            options = self.__createCommandLineOptions(args)
-            return accessory.start(options)
+            return accessory.start(self._create_command_line_options(args))
         return False
 
+    @with_accessories_lock
     def stop(self, name):
-        accessory = self.__accessories[name]
-        if accessory:
+        if accessory := self._accessories[name]:
             return accessory.stop()
         return False
 
+    @with_accessories_lock
     def reboot(self, name):
-        accessory = self.__accessories[name]
-        if accessory:
+        if accessory := self._accessories[name]:
             return accessory.stop() and accessory.start()
         return False
 
-    def factoryResetAll(self):
-        for accessory in self.__accessories.values():
+    @with_accessories_lock
+    def factory_reset_all(self):
+        for accessory in self._accessories.values():
             accessory.factoryReset()
 
-    def factoryReset(self, name):
-        accessory = self.__accessories[name]
-        if accessory:
+    @with_accessories_lock
+    def factory_reset(self, name):
+        if accessory := self._accessories[name]:
             return accessory.factoryReset()
         return False
 
-    def waitForMessage(self, name, message, timeoutInSeconds=10):
-        accessory = self.__accessories[name]
-        if accessory:
+    @with_accessories_lock
+    def wait_for_message(self, name, message, timeoutInSeconds=10):
+        if accessory := self._accessories[name]:
             # The message param comes directly from the sys.argv[2:] of WaitForMessage.py and should contain a list of strings that
             # comprise the entire message to wait for
             return accessory.waitForMessage(' '.join(message), timeoutInSeconds)
         return False
 
-    def createOtaImage(self, otaImageFilePath, rawImageFilePath, rawImageContent, vid='0xDEAD', pid='0xBEEF'):
+    def create_ota_image(self, otaImageFilePath, rawImageFilePath, rawImageContent, vid='0xDEAD', pid='0xBEEF'):
         # Write the raw image content
         with open(rawImageFilePath, 'w') as rawFile:
             rawFile.write(rawImageContent)
 
         # Add an OTA header to the raw file
-        otaImageTool = _DEFAULT_CHIP_ROOT + '/src/app/ota_image_tool.py'
-        cmd = [otaImageTool, 'create', '-v', vid, '-p', pid, '-vn', '2',
+        otaImageTool = _DEFAULT_CHIP_ROOT / 'src/app/ota_image_tool.py'
+        cmd = [str(otaImageTool), 'create', '-v', vid, '-p', pid, '-vn', '2',
                '-vs', "2.0", '-da', 'sha256', rawImageFilePath, otaImageFilePath]
         s = subprocess.Popen(cmd)
         s.wait()
         if s.returncode != 0:
             raise Exception('Cannot create OTA image file')
```

Comment:
Throwing vanilla `Exception` is not recommended. `RuntimeError` should be appropriate here.